.github/workflows/linkcheck.yml

```diff
@@ -17,5 +17,13 @@ jobs:
           key: cache-lychee-${{ github.sha }}
           restore-keys: cache-lychee-
 
+      # accept 429 only for PR runs where secret isn't available
       - name: Check links
-        run: nix run nixpkgs#lychee -- --verbose docs README.md
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        run: |
+          nix run nixpkgs#lychee -- \
+            --verbose \
+            --cache \
+            --accept '${{ github.event_name == 'pull_request' && '200, 429' || '200' }}' \
```

Comment:
Looks like this failed in the merge queue, so we may need to also allow 429s in `merge_group`.

https://github.com/fedimint/fedimint/actions/runs/14821865244/job/41609922008#step:4:220